src/Model/SnapshotInterface.php

```diff
@@ -18,6 +18,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  *
+ * @method int|null           getId()
```

Comment:
Should this be `int`? or `mixed`?

Based on `PageInterface`

https://github.com/sonata-project/SonataPageBundle/blob/d102ce8843f3fb9456ce7d44d3af1194c959e8cb/src/Model/PageInterface.php#L27-L32

(I don't know if TargetId should be `mixed` as well)

Reply:
The `parentId` is a int type at the moment: https://github.com/sonata-project/SonataPageBundle/blob/fed26c06856de8d6b1161d0f23001db7b8377e2b/src/Model/SnapshotInterface.php#L26

If we change this, we should change it everywhere.